src/exodus/wallet.cpp

```diff
@@ -39,7 +39,7 @@ void Wallet::UpdateSigmaMint(
     e.groupId = groupId;
     e.index = index;
     e.block = block;
-    e.isUsed = false;
```

Comment:
Seem like this is a bug. We should not update used status when we update chain state.

Reply:
removed